src/test/java/uk/gov/pay/directdebit/mandate/dao/MandateDaoIT.java

```diff
@@ -274,6 +274,44 @@ public void shouldNotUpdateAnythingIfTransactionDoesNotExist() {
         assertThat(numOfUpdatedMandates, is(0));
     }
 
+    @Test
+    public void shouldUpdateStateByProviderIdAndReturnNumberOfAffectedRows() {
+        GatewayAccountFixture goCardlessGatewayAccountFixture = GatewayAccountFixture.aGatewayAccountFixture()
```

Comment:
The descriptive values really help to understand what all the set up if so  👍 👍 👍

Reply:
I have my moments!